packages/office-ui-fabric-react/src/components/Slider/Slider.scss

```diff
@@ -1,24 +1,129 @@
 @import '../../common/common';
 
-.root {
-  user-select: none;
-  margin-bottom: 8px;
-}
+.row, .column {
+  .root {
```

Comment:
nesting.

Reply:
please fix all of these to not nest

Reply:
actually reading more closesly, these are your shared rules (regardless of orientation.) There is no need to add the row/column prefix for these. These rules are your base and they always apply.